### ISSUE TITLE
Actor_loss negative and increasing with stable_baselines implementation

### ISSUE THREAD

**Issue by csingh27sewts** [csingh27sewts/Masterarbeit#80]

**Comment by csingh27sewts** [comment]
Use same parameters as rlpyt

**Comment by csingh27sewts** [comment]
Multiply by 20 like for custom environment

**Comment by csingh27sewts** [comment]
Something is wrong with the environment file since SAC stable baselines work with it and also the actor loss with cloth environment when using custom environment also increases.

**Comment by csingh27sewts** [comment]
Display all plots - Value, critic, etc. and compare the plots with custom implementation

**Comment by csingh27sewts** [comment]
Problem is with the environment.
Problem could be normalization with the environment states, actions ,rewards
You could test whether the environment performs as expected. It does so in DMControlEnv
But does it after converting to gym ?
Simplify the environment and get results

**Comment by csingh27sewts** [comment]
Create an extremely simplistic experiment and test results with it

**Comment by csingh27sewts** [comment]
actor loss = log_probs - critic value
critic value = reward + next value
reward negative = actor loss is positive

**Comment by csingh27sewts** [comment]
Try dm_control suite simpler environments , e.g. cartpole and see if the results are sensible. If not, define mujoco environment using gym from scratch

**Comment by csingh27sewts** [comment]
Also getting negative loss in cartpole and fish environment defined in DMControl.

**Comment by csingh27sewts** [comment]
But it also occurs in normal gym environment "MountainCarContinuous-vo"

**Comment by csingh27sewts** [comment]
So maybe I just need to run the cloth environment for a very large no. of steps say 500000

**Comment by csingh27sewts** [comment]
But I will try to run with cartpole in dmcontrol suite environment even though it did not coverge after even 50k steps...

**Comment by csingh27sewts** [comment]
But a good point to note is that negative actor loss is not a problem

**Comment by csingh27sewts** [comment]
Acrobot environment defined in DmControlSuite converges even though it starts with negative actor loss.

**Comment by csingh27sewts** [comment]
Pendulum environment defined in DmControlSuite converges even though it starts with negative actor loss.

**Comment by csingh27sewts** [comment]
Will run cloth_corner_minimal for very large no. of steps 5,00,000

**Comment by csingh27sewts** [comment]
It is only an issue related to reward scaling. Even though the actor loss seems to increase, it converges after some steps. And it converges to a large negative value of actor loss. Scale the reward.
